### DISCUSSION TITLE
在 mpv.conf 里调用 input.conf 语法

### DISCUSSION THREAD

**Discussion by natural-harmonia-gropius** [hooke007/mpv_PlayKit#220] [mpv 通用]
https://github.com/Natural-Harmonia-Gropius/InputEvent/issues/20

自己实现条件监听太困难了，我想直接像这样利用条件配置更加自然地实现。
```ini
[end]
profile-cond=eof_reached
profile-restore=copy-equal
script-message-to inputevent bind key json
```

**Comment by hooke007** [comment]
profiles里只让写option=value
script-message-to 这种command没法用的

**Reply by hooke007** [reply]
https://github.com/mpv-player/mpv/pull/13612
还没测试能不能在运行时更改

**Reply by hooke007** [reply]
merged https://github.com/mpv-player/mpv/commit/91489c946689763c36e4d720d6048037252818f7

**Comment by dyphire** [comment]
mpv本身并没有支持这种操作，但你可以参考第三方脚本实现的类似功能：[profile-command.lua](https://github.com/CogentRedTester/mpv-scripts/blob/master/profile-command.lua)
